src/utils/oauth/config.json

```diff
@@ -0,0 +1,27 @@
+{
+  "providers": {
+    "github": {
+      "details": {
+        "dev": {
```

Comment:
Add documentation.
Example: /** @see {@link LINK_TO_ORGANIZATION |Organization} for further information. */

Reply:
Comments are not permitted in config.json!